src/homework/forth/MyArrayList.java

```diff
@@ -0,0 +1,53 @@
+package homework.forth;
+
+public class MyArrayList<T> implements List<T> {
+    private Object[] arr;
+    private int counter;
+
+    public MyArrayList() {
+        arr = new Object[10];
+    }
+
+    public T get(int index) {
+        if (index < 0 || index >= counter) {
+            System.out.println("Error. Please check the index " + index);
+            return null;
+        } else {
+            return (T) arr[index];
+        }
+    }
+
+    public void add(T t) {
+        if (arr.length == counter) {
+            Object[] arrIncreased = new Object[arr.length * 3 / 2];
+            for (int i = 0; i < arr.length; i++) {
+                arrIncreased[i] = arr[i];
+            }
+            arr = arrIncreased;
+        }
+        arr[counter] = t;
+        counter++;
+    }
+
+    public void remove(int index) {
+        if (index < 0 || index >= counter) {
+            System.out.println("Error. Please check the index " + index);
+        } else {
+            for (int j = index; j < counter; j++) {
```

Comment:
The complexity of your algorithm is O(N). Why not use System.arraycopy()? It will help to make complexity O(1).